packages/react-menu/src/components/MenuTrigger/renderMenuTrigger.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { MenuTriggerState } from './MenuTrigger.types';
+
+/**
+ * Render the final JSX of MenuTrigger
+ *
+ * Only renders children
+ * {@docCategory MenuTrigger }
+ */
+export const renderMenuTrigger = (state: MenuTriggerState) => {
+  return <>{state.children}</>;
```

Comment:
```suggestion
  return state.children;
```

`.children` should be `React.ReactElement` in this case as usage of `.Children.only()` (`useMenuTrigger.ts`) will throw if multiple nodes are passed.

Reply:
Interesting, because TS does not like if I return only state.children... I will try to do some TS -fu

Reply:
Thanks for the children typing suggestions